### PR TITLE
fortinet_fortiedr: new package for Fortinet FortiEDR ingestion

### PR DESCRIPTION
**Pull request by efd6**
## What does this PR do?

This adds a package for Fortinet FortiEDR ingestion.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- For #1190

### CODE REVIEW

File: packages/fortinet_fortiedr/_dev/build/build.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+dependencies:
+  ecs:
+    reference: git@v8.3.0
```
Comment (P1llus) [inline packages/fortinet_fortiedr/_dev/build/build.yml:3]: Shall we bump this one?

File: packages/fortinet_fortiedr/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+      field: observer.type
+      value: edr
+  - set:
+      field: event.category
+      value: malware
+  - rename:
+      field: message
+      target_field: event.original
+      ignore_missing: true
+      ignore_failure: true
```
Comment (P1llus) [inline packages/fortinet_fortiedr/data_stream/log/elasticsearch/ingest_pipeline/default.yml:23]: Any reason for needing `ignore_failure`? We usually don't want to have that unless absolutely necessary

File: packages/fortinet_fortiedr/data_stream/log/manifest.yml [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+        title: Add non-ECS fields
+        required: false
+        show_user: true
+        default: true
+      - name: keep_raw_fields
+        type: bool
+        title: Keep raw parser fields
+        required: false
+        show_user: false
+        default: false
```
Comment (P1llus) [inline packages/fortinet_fortiedr/data_stream/log/manifest.yml:188]: Are these maybe from older package?

File: packages/fortinet_fortiedr/manifest.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+title: Fortinet FortiEDR Logs
+version: 1.0.0
+release: ga
+description: Collect logs from Fortinet FortiEDR instances with Elastic Agent.
+type: integration
+format_version: 1.0.0
+license: basic
+categories: ["security"]
+conditions:
+  kibana.version: "^7.14.1 || ^8.0.0"
```
Comment (P1llus) [inline packages/fortinet_fortiedr/manifest.yml:11]: We could bump this to 7.17 maybe? Don't know how far back we want to support this, up to @jamiehynds

Comment (jamiehynds) [reply packages/fortinet_fortiedr/manifest.yml:11]: @P1llus sounds good - supporting 7.17 or later is what we've done with most other new integrations so best to bump FortiEDR to 7.17 too.

File: packages/fortinet_fortiedr/data_stream/log/manifest.yml (hunk 3 above)
Comment (efd6) [reply packages/fortinet_fortiedr/data_stream/log/manifest.yml:188]: Yeah, these were residual in fortinet_fortimanager. I'll drop them.

File: packages/fortinet_fortiedr/_dev/build/build.yml (hunk 1 above)
Comment (efd6) [reply packages/fortinet_fortiedr/_dev/build/build.yml:3]: Done

File: packages/fortinet_fortiedr/manifest.yml (hunk 4 above)
Comment (efd6) [reply packages/fortinet_fortiedr/manifest.yml:11]: Done

**Review by P1llus — APPROVED** [review]

**Comment by jamiehynds** [comment]
@efd6 I'm a day late and a dollar short, but Fortinet have just provided me with an NFR license for FortiEDR. If it's still of use for integration testing, I can provide the details.

**Comment by efd6** [comment]
@jamiehynds Thank. That will be useful for the dashboard.

**Pull request merged**